commit f84a66d61e1f12fc7eba169d78885a060c04d00b
Author: Hans Verkuil <hans.verkuil@cisco.com>
Date:   Thu Feb 8 11:25:15 2018 -0500

    media: cec-pin-error-inj.rst: document CEC Pin Error Injection
    
    The CEC Pin framework adds support for Error Injection.
    
    Document all the error injections commands and how to use it.
    
    Signed-off-by: Hans Verkuil <hans.verkuil@cisco.com>
    Signed-off-by: Mauro Carvalho Chehab <mchehab@s-opensource.com>

diff --git a/Documentation/media/uapi/cec/cec-api.rst b/Documentation/media/uapi/cec/cec-api.rst
index b68ca9c1d2e0..1e2cf498ba30 100644
--- a/Documentation/media/uapi/cec/cec-api.rst
+++ b/Documentation/media/uapi/cec/cec-api.rst
@@ -23,6 +23,7 @@ This part describes the CEC: Consumer Electronics Control
 
     cec-intro
     cec-funcs
+    cec-pin-error-inj
     cec-header